robotics/actuator/esc_group/TorqueSetpoint.0.1.uavcan

```diff
@@ -0,0 +1,9 @@
+#
+# Setpoints for the torque control loop of an ESC.
+# Each setpoint value contains the target torque applied to the mechanical load in newton*meters.
+#
+
+void2
+uavcan.si.unit.torque.Scalar.1.0[<64] motor_torque
+
+@assert _offset_ % 8 == {0}
```

Comment:
I would change the torque setpoint to the current setpoint. There are two main reasons:
1. The most ESCs have current control loop. And the controller can precisely control this value. In the case of torque control, the precise values of motor parameters (magnetic flux for example) are required. And the ESC can't control the torque precisely. Moreover, not all controllers support the identification of all the necessary motor parameters.
2. Torque and current are not always equally related. In the field weakening, for example, the torque doesn't depend only on the quadrature current. For motors with an independent flux control scheme, the torque can't be determined  from the current directly as well.

Reply:
Fixed by removal.

Reply:
But what about the current setpoint?  
Although, I would agree that the current setpoint can be only ratiometric.

Reply:
I would argue for the torque setpoint message because it is logically more abstract, allowing the designer of the control system to isolate the parameters of the propulsion system (by setting the torque directly, you don't care about the electrical motor constant, for example), but I think that at this point we may not have the required empirical data to make a sensible judgment. My initial proposal was inspired by the design of the AutoQuad's proprietary Quatos attitude stabilization algorithm, which I believed was based on a torque control loop, but upon a closer look it turned out that they actually command mechanical RPM, so the only justification for this message turned out to be invalid.

I think in the future we should consider defining a ***power*** setpoint message instead, because it insulates the setpoint generation logic from the specifics of the motor, and permits the motor controller to transform that into armature/quadrature current in a more or less straightforward way. Field weakening may complicate things though, but I am not sure those specifics should be exposed at the abstract protocol level.

For now, it is safe to just omit it and focus on more important things, such as the BMS message.